src/modules/Library/features/Popups/AddToBuilderPopup/AddToBuilderPopup.tsx

```diff
@@ -37,21 +38,17 @@ export const AddToBuilderPopup = ({
   const isOnline = useNetwork();
   const handleClearCart = useClearCart();
   const [applets, setApplets] = useState<Applet[]>([]);
-  const workspacesWithRoles = workspacesState.useWorkspacesRolesData();
```

Comment:
could you pls take a look and clean if it's needed Workspaces global state from unnecessary code if you use useAsync instead?